spec/spec.md

```diff
@@ -0,0 +1,180 @@
+<!--
+SPDX-Copyright: 2019 Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# REUSE Specification
+
+This specification defines a standardized method for declaring copyright and
+licensing for software projects. The goal of the specification is to have
+unambiguous, machine-readable copyright and licensing information for each
+individual file in a project.
+
```

Comment:
I'm sorry I don't understand well enough what you mean here.

Reply:
I think @silverhook means that there is a big benefit of having copyright information **in** the files, not somewhere else like a DEP-5 file, because they will retain even if just a single file is copied from a project. Right?

Reply:
> I think @silverhook means that there is a big benefit of having copyright information **in** the files, not somewhere else like a DEP-5 file, because they will retain even if just a single file is copied from a project. Right?

Yes, that’s exactly what I meant with that suggestion.